### ISSUE TITLE
Move /etc/zabbix/libzbxpgsql.conf under /etc/zabbix/zabbix_agentd.d/

### ISSUE THREAD

**Issue by mdfod365** [cavaliercoder/libzbxpgsql#116]
It would be great if we could specify the `DEFAULT_PGCONFIGFILE` at "runtime" in the _/etc/zabbix/zabbix_agentd.d/libzbxpgsql.conf_ file, instead of specifying statically when compiling.

Something like:

```
LoadModule=libzbxpgsql.so
ConfFile=<some_path>/libzbxpgsql.conf
```

In this way we could move the actual _/etc/zabbix/libzbxpgsql.conf_ into another directory and keep /etc/zabbix/ only with the official configuration files.

If this is too difficult, it would be still better to change the default path from _/etc/zabbix/libzbxpgsql.conf_ to _/etc/zabbix/zabbix_agentd.d/libzbxpgsql.d/libzbxpgsql.conf_

In conclusion, any third-party configuration file should stay in the _zabbix_agentd,d_ directory and not in the _/etc/zabbix_ top directory.

Thank you.
